doc: add snippet for k8s objects count

rverchere opened the pull request.

rverchere committed:
doc: add snippet for k8s objects count

Signed-off-by: Rémi Verchère <remi@verchere.fr>

scraly commented on pages/platform/kubernetes-k8s/etcd-quota-error/guide.de-de.md:
```diff
@@ -154,6 +154,28 @@ kubectl get reportchangerequest.kyverno.io -A | wc -l
 
+## Counting all resources
+
+If you still need to check all resources as you do not know what consumes etcd quotas, you can run this snippet. You will need the `count` plugin for `kubectl`:
```
How can I install this plugin? ;-)
Through krew?
Please provide the way to install it 

thanks :)

rverchere replied:
Yes you need to install it with krew. I'll update it.

rverchere replied:
Added link to the installation instead of complete how-to install plugin+krew which is not revelant here.

scraly reviewed (changes requested):
How can I install this plugin? ;-)
Through krew?
Please provide the way to install it 

thanks :)

rverchere committed:
doc: add installation link for count k8s plugin

Signed-off-by: Rémi Verchère <remi@verchere.fr>

OVH-Team-Guides commented:
Hello @rverchere, could you please edit the "updated:" field in each guide's header? 
So that it is `updated: 2023-06-29`
Many thanks

rverchere commented:
> Hello @rverchere, could you please edit the "updated:" field in each guide's header? So that it is `updated: 2023-06-29` Many thanks

Hi, I've already updated to 2023-06-28, which is the date i modified the files. See https://github.com/ovh/docs/pull/4857/commits/6543ae527ac08dd0cd15a380067fdacc1caf4c72#diff-bf0947af24bf0df197a1f085a237e1395f37429f8c011f7f31bb6c8343dae270L6

OVH-Team-Guides commented:
Thank you for your contribution @rverchere

scraly commented:
The command is not working for mac users (and maybe more?).
We are in a discussion with @rverchere to fix this.
